### DISCUSSION TITLE
数据权限拦截未生效

### DISCUSSION THREAD

**Discussion by zwac00001** [zuihou/zuihou-admin-cloud#288] [01.常见问题]
跟踪代码进入MybatisPlusInterceptor intercept 方法，发现InnerInterceptor 没有DataScopeInnerInterceptor的实现

**Comment by zuihou** [comment]
先查看：  https://tangyh.top/doc/advanced/authority/%E9%85%8D%E7%BD%AE%E6%95%B0%E6%8D%AE.html#%E6%9D%83%E9%99%90%E6%8E%A7%E5%88%B6
